examples/basic/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "basic"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+ovmf-prebuilt = "0.2.4"
+
+[workspace]
+resolver = "3"
+members = ["kernel"]
+
+[build-dependencies]
+kernel = { path = "kernel", artifact = "bin", target = "x86_64-unknown-none" }
+bootloader = "0.11.12"
```

```suggestion
[workspace]
resolver = "3"
members = ["kernel"]

[package]
name = "basic"
version = "0.1.0"
edition = "2024"

[dependencies]
ovmf-prebuilt = "0.2.4"

[build-dependencies]
kernel = { path = "kernel", artifact = "bin", target = "x86_64-unknown-none" }
bootloader = "0.11.12"
```

Let separate the sections for the workspace and the top-level crate.